src/gt.ts

```diff
@@ -0,0 +1,23 @@
+import curry from "./curry";
+import Curry from "./types/Curry";
+
+/**
+ * Make sure that it is less than the reference value.
+ * It does not provide a generic type, so we are going to provide curry by default.
+ *
+ * @example
+ * ```ts
+ * filter(gt(5), [1, 2, 4, 5, 8, 9]) // Iterable<[8, 9]>
+ * filter(gt(5), [1, 2, 3, 4, 5]) // Iterable<[]>
+ * filter(gt("b"), ["a", "b", "c"]) // Iterable<["c"]>
+ * filter(gt("b"), ["a", "b"]) // Itreable<[]>
+ * ```
+ */
+function gt(a: string): Curry<(b: string) => boolean>;
+function gt(a: number): Curry<(b: number) => boolean>;
```

Comment:
I think the signature below should also be provided.
```ts
function gt(a: number, b:number): boolean;
```

Reply:
I don't understand

I understood this.
Is this right?

```ts
function gt(a: number, b: number) : boolean { return a > b; }

const result = gt(5, 1); // expected true
```